kyuubi-server/src/main/scala/org/apache/kyuubi/events/KyuubiServerInfoEvent.scala

```diff
@@ -0,0 +1,83 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kyuubi.events
+
+import org.apache.kyuubi
+
+import org.apache.kyuubi.Utils
+import org.apache.kyuubi.server.KyuubiServer
+import org.apache.kyuubi.service.ServiceState
+
+/**
+ * A [[KyuubiServerInfoEvent]] is used to get server level info when KyuubiServer start or stop
+ *
+ * @param serverName the server name
+ * @param startTime the time when the server started
+ * @param eventTime the time when the event made
+ * @param states the server states
+ * @param serverIP the server ip
+ * @param serverConf the server config
+ * @param serverEnv the server environment
+ * @param extraStatesInfo the server extra explain of states
+ */
+case class KyuubiServerInfoEvent private(
+    serverName: String,
+    startTime: Long,
+    eventTime: Long,
+    states: String,
+    serverIP: String,
+    serverConf: Map[String, String],
+    serverEnv: Map[String, String],
+    extraStatesInfo: String) extends KyuubiServerEvent {
```

Comment:
we don't need this field, `state` is enough.

Reply:
State is always started. That because only server is started that we can log event. So I think we need more information to help judge the server status. Do you have a better way to resolve it?

Reply:
we can hard code `state` to `stopped`, and same with `started` so we don't need `extraStatesInfo `.

Reply:
Got it.